docs/reference/tctl/workflow.md

```diff
@@ -7,121 +7,28 @@ tags:
   - tctl
 ---
 
-<!-- prettier-ignore -->
-import * as WhatIsAWorkflowExecution from '../../content/what-is-a-workflow-execution.md'
-
-The `tctl workflow` commands operate <preview page={WhatIsAWorkflowExecution}>Workflow Executions</preview>.
-
-## `show`
-
-import TctlWorkflowShow from './workflow/show.md'
-
-<TctlWorkflowShow/>
-
-## `showid`
-
-import TctlWorkflowShowid from './workflow/showid.md'
-
-<TctlWorkflowShowid/>
-
-## `start`
-
-import TctlWorkflowStart from './workflow/start.md'
-
-<TctlWorkflowStart/>
-
-## `run`
-
-import TctlWorkflowRun from './workflow/run.md'
-
-<TctlWorkflowRun/>
-
-## `cancel`
-
-import TctlWorkflowCancel from './workflow/cancel.md'
-
-<TctlWorkflowCancel/>
-
-## `signal`
-
-import TctlWorkflowSignal from './workflow/signal.md'
-
-<TctlWorkflowSignal/>
-
-## `terminate`
-
-import TctlWorkflowTerminate from './workflow/terminate.md'
-
-<TctlWorkflowTerminate/>
-
-## `list`
-
-import TctlWorkflowList from './workflow/list.md'
-
-<TctlWorkflowList/>
-
-## `listall`
-
-import TctlWorkflowListall from './workflow/listall.md'
-
-<TctlWorkflowListall/>
-
-## `listarchived`
-
-import TctlWorkflowListarchived from './workflow/listarchived.md'
-
-<TctlWorkflowListarchived/>
-
-## `scan`
-
-import TctlWorkflowScan from './workflow/scan.md'
-
-<TctlWorkflowScan/>
-
-## `count`
-
-import TctlWorkflowCount from './workflow/count.md'
-
-<TctlWorkflowCount/>
-
-## `query`
-
-import TctlWorkflowQuery from './workflow/query.md'
-
-<TctlWorkflowQuery/>
-
-## `stack`
-
-import TctlWorkflowStack from './workflow/stack.md'
-
-<TctlWorkflowStack/>
-
-## `describe`
-
-import TctlWorkflowDescribe from './workflow/describe.md'
-
-<TctlWorkflowDescribe/>
-
-## `describeid`
-
-import TctlWorkflowDescribeid from './workflow/describeid.md'
-
-<TctlWorkflowDescribeid/>
-
-## `observe`
-
-import TctlWorkflowObserve from './workflow/observe.md'
-
-<TctlWorkflowObserve/>
-
-## `reset`
-
-import TctlWorkflowReset from './workflow/reset.md'
-
-<TctlWorkflowReset/>
-
-## `reset-batch`
-
-import TctlWorkflowResetbatch from './workflow/reset-batch.md'
-
-<TctlWorkflowResetbatch/>
+The `tctl workflow` commands enable [Workflow Execution](/docs/content/what-is-a-workflow-execution) operations.
+
+- [`tctl workflow show`](./workflow/show)
```

Comment:
```suggestion
- [`tctl workflow show`](./workflow/show)
```

For consistency and predicability - lets use the full path from the project root, i.e. `/docs/reference/tctl/workflow/show`